qubekit/fragmentation/WBOFragmenter.py

```diff
@@ -0,0 +1,228 @@
+        """
+        # sanitize is off to keep the "atom_map" property
+        rdmol = Chem.MolFromSmiles(semi_mapped_smile, sanitize=False)
+
+        # add at least one conformer
+        Chem.AllChem.EmbedMolecule(rdmol)
+
+        # Chem.SanitizeMol calls updatePropertyCache so we don't need to call it ourselves
+        # https://www.rdkit.org/docs/cppapi/namespaceRDKit_1_1MolOps.html#a8d831787aaf2d65d9920c37b25b476f5
+        Chem.SanitizeMol(
+            rdmol,
+            Chem.SANITIZE_ALL ^ Chem.SANITIZE_ADJUSTHS ^ Chem.SANITIZE_SETAROMATICITY,
+        )
+        Chem.SetAromaticity(rdmol, Chem.AromaticityModel.AROMATICITY_MDL)
+
+        # Chem.MolFromSmiles adds bond directions (i.e. ENDDOWNRIGHT/ENDUPRIGHT), but
+        # doesn't set bond.GetStereo(). We need to call AssignStereochemistry for that.
+        Chem.AssignStereochemistry(rdmol)
+
+        # convert the rdmol to the qubekit Ligand
+        molecule = Ligand.from_rdkit(rdmol, name)
+
+        # attach the fragments as their own ligands
+        if bond_indices is not None:
+            molecule.bond_indices = bond_indices
+
+        # ensure that the atoms are ordered the same way
+        for atom, rdkit_atom in zip(molecule.atoms, rdmol.GetAtoms()):
+            if not rdkit_atom.GetAtomicNum() == atom.atomic_number:
+                raise Exception(f'Ligand created from an rdkit molecule should have the same atom order: rdkit {rdkit_atom.GetAtomMapNum()}, ligand {atom.atomic_number}')
+            atom.map_index = rdkit_atom.GetAtomMapNum()
+
+        return molecule
+
+    def _results_to_ligand(self, fragmentation):
+        """
+        TODO
+        """
+        # copy the fragmentation-relevant indices
+        molecule = self._create_ligand(fragmentation.parent_smiles, 'molecule')
+
+        molecule.fragments = []
+        for fragment in fragmentation.fragments:
+            # partly verify that the fragments can still be mapped to the parent molecule
+            atoms = [a for a in molecule.atoms if a.map_index in fragment.bond_indices]
+            if len(atoms) != 2:
+                raise Exception('The index used in the fragment to map to the parent could not be found in the parent.')
+
+            # copy the fragmentation-relevant indices
+            fragment_molecule = self._create_ligand(fragment.smiles, 'fragment', bond_indices=fragment.bond_indices)
+
+            molecule.fragments.append(fragment_molecule)
+
+        return molecule
+
+    def _run(self, molecule: "Ligand", *args, **kwargs) -> "Ligand":
+        return self.run(molecule, *args, **kwargs)
+
+    def run(self, molecule: Ligand, **kwargs) -> Ligand:
+        from openff.fragmenter.depiction import depict_fragmentation_result
+        from openff.fragmenter.fragment import WBOFragmenter
+
+        # convert to an OpenFF Molecule (canonical order)
+        off_molecule: OFFMolecule = self._prepare_fragment_input(molecule)
+
+        fragmentation = WBOFragmenter().fragment(
```

Comment:
Do we want to expose any of the other settings for the WBOFragmenter like a threshold or to keep non-rotor groups?